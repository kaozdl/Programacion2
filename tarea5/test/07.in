# Caso 07
ingresar_localidades (15,ulsan) (41,tambo) (70,goya) (100,esmeraldas) (60,cumana) .
iniciar_pedidos 5
ingresar_pedidos ulsan (1000,chapas) 5 ulsan (2000,comestibles) 3 .
ingresar_pedidos ulsan (1000,medicamentos) 5 ulsan (2000,medicamentos) 3 .
ingresar_pedidos ulsan (1000,chapas) 1 ulsan (2000,comestibles) 4 .
imprimir_totales
ingresar_pedidos tambo (1200,chapas) 5 tambo (1100,chapas) 4 tambo (1000,chapas) 3 tambo (900,chapas) 2 tambo (800,chapas) 1 .
procesar_rango 2
procesar_rango 1
procesar_rango 1
procesar_rango 3
ingresar_pedidos esmeraldas (1200,chapas) 5 tambo (1100,chapas) 4 tambo (1000,chapas) 3 tambo (900,chapas) 2 esmeraldas (800,chapas) 1 .
imprimir_totales
Fin
