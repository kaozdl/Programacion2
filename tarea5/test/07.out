1>2>3>
 IDENTIFICADORES DE LOCALIDADES 
(1,cumana)(2,esmeraldas)(3,goya)(4,tambo)(5,ulsan)
4>5>6>7>INFORME POR RANGOS
9000
0
0
0
0
8>9>10>ulsan
(1000,chapas)(2000,comestibles)(2000,medicamentos)(2000,comestibles)(1000,chapas)(1000,medicamentos)
11>ulsan

12>cumana

tambo
(800,chapas)(900,chapas)(1000,chapas)(1100,chapas)(1200,chapas)
13>14>INFORME POR RANGOS
0
0
3000
0
2000
15>Fin.
